cpio/bsdcpio.1

```diff
@@ -50,7 +50,7 @@
 .Nm
 copies files between archives and directories.
 This implementation can extract from tar, pax, cpio, zip, jar, ar,
-and ISO 9660 cdrom images and can create tar, pax, cpio, ar,
+and ISO 9660 cdrom images and can create tar, cpio, ar,
```

Comment:
Please put back the reference to `pax` format, which is supported by bsdcpio.

Reply:
@kientzle Kindly please notice that `pax` is not available as an option https://github.com/libarchive/libarchive/blob/01a280fbf0794f575eb39d718a558896add18d30/cpio/cpio.c#L472

Reply:
We should update the help message also, it seems.  In reality, that option allows any format supported by libarchive:
```
	r = archive_write_set_format_by_name(cpio->archive, cpio->format);
```